src/locale/dim.json

```diff
@@ -310,20 +310,6 @@
     "WishlistDupe": "Shows duplicate items where at least one of the duplicates is on your wish list.",
     "Year": "Shows items from which year of Destiny they appeared in."
   },
-  "Ghost": {
-    "crucible": "crucible",
-    "dreaming": "dreaming",
-    "edz": "edz",
-    "gambit": "gambit",
-    "io": "io",
-    "leviathan": "leviathan",
-    "mars": "mars",
-    "mercury": "mercury",
-    "nessus": "nessus",
-    "strikes": "strikes",
-    "tangled": "tangled",
-    "titan": "titan"
```

Comment:
Are these really unused? I think they come from `BadgeInfo.tsx#118`.

Reply:
they do. @delphiactual, was dim.json being pruned manually?
that why the strings lasted as long as they did?

the comment-each-string-to-preserve thing feels a little silly,
is there any `Tags.*` and `Ghost.*` option, or an .ignore we have available?

Reply:
alternately would i18n pick up "Ghost.crucible" in a json file?

Reply:
yes pruned manually do to getting into an infinite loop prior to this change.

irt comment-each-string-to-preserve that only happens if we interpolate values or hide them from the scanner, (we can try to make a custom function for locating them, i was never successful) https://github.com/i18next/i18next-scanner#func

it would need to be able to parse something like
`${t(compact ? 'Countdown.DaysCompact' : 'Countdown.Days', { count: days })} ${text}`
and get `t('Countdown.DaysCompact')` and `t('Coundown.Days', { count: days })`
the other rely on context which I do not know if we can provide.

Reply:
we do not comment each string just ones that are not in the most basic format